### PR TITLE
Update readme.md

### PR DESCRIPTION
**Pull request by sindresorhus**
List

- [ ] Unicorn
- [ ] Rainbow

### CODE REVIEW

File: readme.md [hunk 1]
```diff
@@ -1 +1,13 @@
-🦄🦄🦄🦄
+🦄🦄🦄
+
+
+🦄🦄🦄
```
Comment (sindresorhus) [inline readme.md]: asdsadas

File: readme.md [hunk 2]
```diff
@@ -1 +1,13 @@
...
+
+
+
+🦄🦄🦄
```
Comment (sindresorhus) [inline readme.md]: dsfdsf

File: readme.md [hunk 3]
```diff
@@ -1 +1,13 @@
...
+
+
+
```
Comment (sindresorhus) [inline readme.md]: sdfsd

File: readme.md [hunk 4]
```diff
@@ -1 +1,13 @@
-🦄🦄🦄🦄
+🦄🦄🦄
+
+
+🦄🦄🦄
+
+
+
```
Comment (sindresorhus) [inline readme.md]: fsdfds

**Comment by resynth1943** [comment]
we should merge this

**Pull request merged**